### PR TITLE
Uploading stage plots

### PR DESCRIPTION
**Pull request by binary1230**
This has two main feature additions (plus a little code cleanup):
- Stage plots can now be uploaded.  Chaney needs to tell me what words to put to explain what we want, but I added some reasonable sounding text for now.
- Bands can be linked to events on the schedule, which will be useful when we implement the "Final Confirmation" step.

### CODE REVIEW

File: bands/site_sections/band_admin.py [hunk 1]
```diff
@@ -12,6 +12,9 @@ def index(self, session, message=''):
     @ajax
     def mark_as_band(self, session, group_id):
         group = session.group(group_id)
+        if not group.leader:
+            return {'message': '{} does not have an assigned group leader'.format(group.name)}
```
Comment (EliAndrewC) [inline bands/site_sections/band_admin.py]: Certain steps would error in the absence of a group leader, plus there would be no one to receive an email with the link to the page in the first place.

**Comment by EliAndrewC** [comment]
This involved a database change, but since there's no data in the ``band`` table yet other than a single test row, I'll delete that table and let the code re-create it, testing on staging first.  (I'll also double check in production that there is indeed only the one test row.)

**Comment by binary1230** [comment]
LGTM +1
btw, sidenote outside the scope of this change is the question of: should we be storing these images as binary data in the database itself?

Mostly, thinking about it from a backup / deploy perspective.  In the cases when we: move the prod DB from the cloud to a physical server, do any kind of replication magic, or have multiple frontends.  We'll have to rethink our storage/backup strategy to deal with this binary data.

I personally think it'd be cool to have these as binary blobs in the DB, so we only need to worry about backing up one thing when we back up / replicate / move "the data".  I don't know of any performance implications here, though I imagine for most of the images/etc this is purely behind the scenes internal data access that is probably immune from any kind of real performance concerns.

some discussion: http://stackoverflow.com/questions/54500/storing-images-in-postgresql

**Comment by binary1230** [comment]
i put some discussion about this idea in another thread here: https://github.com/magfest/ubersystem/issues/1462

**Comment by EliAndrewC** [comment]
Yeah, I'm definitely not worried about performance in these cases; even if we were doing the dumbest possible implementation and loading the entire file into memory in a way that made multiple in-memory copies of it, I doubt it would affect anything noticeably.  And we don't actually have to do that, since SQLAlchemy supports much smarter ways of streaming the data, and CherryPy will serve anything you give it that implements the file interface.

I generally store files on the filesystem for the sake of a marginally simpler optimization, but storing them in the database probably would be better for the sake of a database backup/replication having everything.

**Pull request merged**